syntax = "proto3";

package bookstore;



service SuggestionsService {
    // recommend similar books based on the book name
    rpc SuggestBooks(BookRequest) returns (SuggestionsResponse);    
}

message BookRequest {
    string book_name = 1;
}

message SuggestionsResponse {
 repeated Suggestions suggestions = 1;
}

message Suggestions {
    string book_title = 1;
    string book_author = 2;
}

